### ISSUE TITLE
Error about window being too small?

### ISSUE THREAD

**Issue by ThomasMarx92** [hood/popui.nvim#12]
Generally popui works fine with all my files and I am happy about it!
In my .tex document however, where I have >100 warnings (chktex linting is very sensitive) I get the following error:

```
E5108: Error executing lua .../.local/share/nvim/plugged/popui.nvim/lua/popui/core.lua:54: unable to create the config, your window is too
small, please zoom out
stack traceback:

[C]: in function 'error'                                          .../.local/share/nvim/plugged/popui.nvim/lua/popui/core.lua:54: in function 'validatePopupSize'                                            .../.local/share/nvim/plugged/popui.nvim/lua/popui/core.lua:71: in function 'getListWindowConfiguration'                          .../.local/share/nvim/plugged/popui.nvim/lua/popui/core.lua:399: in function 'spawnListPopup'                                                ...m/plugged/popui.nvim/lua/popui/diagnostics-navigator.lua:11: in function <...m/plugged/popui.nvim/lua/popui/diagnostics-navigator.lua:3> [string ":lua"]:1: in main chunk
```

Zooming out with <Ctrl> + <Alt> +  "-" does not help.
Any suggestions besides having less warnings?

**Comment by hood** [comment]
Yes, as a user myself I get the same error very often and I'm planning to fix it ASAP. It's basically caused by lines' lengths exceeding the current window's width. I should probably disable this invariant and let the content overflow, while clipping the popup width (same goes for height).

**Comment by hood** [comment]
It should've been fixed by 6017d97c1db22af96b14094e59fc6c45941ddf0d, let me know if it works for you so I can close this issue. 😄

**Comment by ThomasMarx92** [comment]
Perfect! :)
